app/routes/companyInterest/CompanyInterestListRoute.ts

```diff
@@ -40,18 +45,32 @@ const mapStateToProps = (state, props) => {
           })
         : 'Vis alle semestre',
   };
+  const selectedEventOption = {
+    value: eventValue ? eventValue : '',
+    label: eventValue
+      ? EVENT_TYPE_OPTIONS.filter(
```

Comment:
This is knitpickery, but if I understand this correctly, you're fetching the first (and only?) item matching the criteria? Wouldn't it be more intuitive to use `find()`? Like so:
```
EVENT_TYPE_OPTIONS.find((eventType) => eventType.value === eventValue).label
```

Reply:
Much better. Thanks.